src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -9,162 +9,197 @@ const wrapper = createWrapper().findAppLayout();
+              await page.keys(['Tab', 'Right', 'Tab', 'Tab', 'Enter']);
+              await expect(page.isFocused(wrapper.findSplitPanel().findPreferencesButton().toSelector())).resolves.toBe(
+                true
+              );
+            },
+            { pageName: 'with-split-panel', visualRefresh, mobile, splitPanelPosition: 'bottom' }
+          )
+        );
+
+        test(
+          'focuses split panel preferences button when its position changes from side to bottom',
+          setupTest(
+            async page => {
+              await page.click(wrapper.findSplitPanel().findOpenButton().toSelector());
+              await page.click(wrapper.findSplitPanel().findPreferencesButton().toSelector());
+              await page.keys(['Tab', 'Left', 'Tab', 'Tab', 'Enter']);
+              await expect(page.isFocused(wrapper.findSplitPanel().findPreferencesButton().toSelector())).resolves.toBe(
+                true
+              );
+            },
+            { pageName: 'with-split-panel', visualRefresh, mobile, splitPanelPosition: 'side' }
+          )
+        );
 
-    test(
-      'focuses split panel preferences button when its position changes from side to bottom',
-      setupTest(
-        async page => {
-          await page.click(wrapper.findSplitPanel().findOpenButton().toSelector());
-          await page.click(wrapper.findSplitPanel().findPreferencesButton().toSelector());
-          await page.keys(['Tab', 'Left', 'Tab', 'Tab', 'Enter']);
-          await expect(page.isFocused(wrapper.findSplitPanel().findPreferencesButton().toSelector())).resolves.toBe(
-            true
+        test(
+          'does not focus split panel when opening programatically',
+          setupTest(
+            async page => {
+              const selection = wrapper.findContentRegion().findTable().findRowSelectionArea(1).toSelector();
+              await page.click(selection);
+              await expect(page.isFocused(selection + ' input')).resolves.toBe(true);
+            },
+            {
+              pageName: 'with-table-and-split-panel',
+              visualRefresh,
+              mobile,
+            }
+          )
+        );
+
+        describe('focus interaction with info links', () => {
+          test(
+            'moves focus to close button when panel is opened from info link',
+            setupTest(
+              async page => {
+                await page.click(wrapper.findContentRegion().findLink('[data-testid="info-link-1"]').toSelector());
+                await expect(page.isFocused(wrapper.findToolsClose().toSelector())).resolves.toBe(true);
+              },
+              { pageName: 'with-fixed-header-footer', visualRefresh, mobile }
+            )
           );
-        },
-        { pageName: 'with-split-panel', visualRefresh, splitPanelPosition: 'side' }
-      )
+          // tests not relevant for mobile, as panel overlays content
+          if (!mobile) {
```

Comment:
If tests are not relevant for mobile, they can simply live outside the `forEach(mobile => {..})` block.

Personally I find it better to read than keeping in mind that `mobile` var is always `false` down in this condition